invenio_app_ils/ui/src/invenio_app_ils/common/api/stats/stats.js

```diff
@@ -41,8 +41,22 @@ const getMostLoanedDocuments = async (fromDate, toDate) => {
   return response;
 };
 
+const recordStats = async (pidType, pidValue) => {
+  const data = {
+    views: {
+      stat: 'record-view',
+      params: {
+        pid_type: pidType,
+        pid_value: pidValue,
+      },
+    },
+  };
+  return await http.post('/stats', data);
```

Comment:
which url is the correct one ? `circulation/stats/` or `/stats`?

Reply:
Well, it might be not super clear :)
This `stats.js` contains multiple endpoints for stats, not only `circulation` stats. Or we split it in 2, e.g. `circulation_stats.js` and `records_stats.js` or keep everything there.
For now, I put record stats together with circulation stats, as they are stats :)
So, for `most-loaned`, it is `/circulation/stats` (maybe we should have `/circulation/loans/stats`?) and for record, `invenio-stats` registers the views under `/stats`.

Reply:
I see, maybe just a small comment would help to tell which is which (but it is NIT)

Reply:
Probably it is better to split then... I will leave it to Zach :)